py/desispec/workflow/procfuncs.py

```diff
@@ -481,7 +480,7 @@ def submit_batch_script(prow, dry_run=0, reservation=None, strictly_successful=F
     if dry_run:
         ## in dry_run, mock Slurm ID's are generated using CPU seconds. Wait one second so we have unique ID's
         current_qid = int(time.time() - 1.6e9)
-        time.sleep(1)
+        time.sleep(0.1)
```

Comment:
As the comment mentions, this needs to wait one second because the mock qid's are tied to a clock (convenient way to get monotonically increasing and unique values). 

The other change in workflow/utils is welcomed to speed up the tests, but this particular line is a limitation with the current means of simulating submissions and needs to remain.

Reply:
good catch.  I clearly made the update without reading the comment two lines above.  :)  I'll revert.